src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.EcDsa.ImportExport.cs

```diff
@@ -0,0 +1,334 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using Microsoft.Win32.SafeHandles;
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcKeyCreateByKeyParameters", CharSet = CharSet.Ansi)]
+        private static extern int EcKeyCreateByKeyParameters(
+            out SafeEcKeyHandle key,
+            string oid,
+            byte[]? qx, int qxLength,
+            byte[]? qy, int qyLength,
+            byte[]? d, int dLength);
+
+        internal static SafeEcKeyHandle EcKeyCreateByKeyParameters(
+            string oid,
+            byte[]? qx, int qxLength,
+            byte[]? qy, int qyLength,
+            byte[]? d, int dLength)
+        {
+            SafeEcKeyHandle key;
+            int rc = EcKeyCreateByKeyParameters(out key, oid, qx, qxLength, qy, qyLength, d, dLength);
+            if (rc == -1)
+            {
+                key?.Dispose();
+                Interop.Crypto.ErrClearError();
```

Comment:
I'd remove all the `ErrClearError` calls. The implementation in AndroidCrypto is empty (we should just remove it as we keep splitting off).